app/assets/stylesheets/application.css

```diff
@@ -13,3 +13,57 @@
  *= require_tree .
  *= require_self
  */
+
```

Comment:
![image](https://github.com/kkolade/rails_blog_app/assets/93039359/254aad45-447d-49bd-bd9b-584f23439621)

- [ ] It would be nice if you fix the `Live Demo`, `Deployment` and `FAQ` links because these links are broken :fearful: Kindly fix them or remove them to make your readme more descriptive and professional :+1: